crates/api_crud/src/comment/create.rs

```diff
@@ -134,6 +134,7 @@ pub async fn create_comment(
     &local_user_view.person,
     true,
     &context,
+    Some(&local_user_view),
```

Comment:
Do this for the other 3 api_crud files instead of using `None` in those.

Reply:
Done